addons/account/models/account_move_line.py

```diff
@@ -441,6 +441,15 @@ class AccountMoveLine(models.Model):
     # === Misc Information === #
     is_refund = fields.Boolean(compute='_compute_is_refund')
 
+    no_followup = fields.Boolean(
+        string="No Follow-Up",
+        compute='_compute_no_followup',
+        inverse='_inverse_no_followup',
+        store=True,
```

Comment:
Why is it that one that is stored, and not the one on the move ? :thinking: 
Stored with inverse function seems weird :monocle_face: